src/components/picker/PickerItemsList.tsx

```diff
@@ -0,0 +1,96 @@
+// TODO: This should be renamed to PickerOverlayContent cause it's
```

Comment:
Why is this TODO not included in this refactor? What's holding it back?

Reply:
Already renamed to PickerItemsList instead, removed the TODO